Added remove_unmapped_handles_from_rank_file function.

souravtecken opened the pull request:
1. Resolves #15

souravtecken committed:
Added remove_unmapped_handles_from_rank_file function.
1. Resolves #15

varunvora commented on database/db_tools.py:
```diff
@@ -152,17 +152,19 @@ def remove_unmapped_handles_from_rank_file(file_name: str) -> None:
+
+    with open(CONTEST_RANKS_DIR,file_name,'w') as rank_file:
+        for user_name_line in input_data.split("\n"):
+            check_occurence_in_line=0
```
Use boolean and mention datatype (mypy)

souravtecken committed:
Merge branch 'master' of https://github.com/varunvora/alcoding

souravtecken committed:
Merge branch 'master' of https://github.com/varunvora/alcoding

souravtecken committed:
Updated April-Easy 2019 rankints.
1. Used boolean and mentioned datatype - remove_unmapped_handles_from_rank_file function.
2. Updated rankings for Hackerearth-April-Easy 2019 ranklists.
3. Added codejam handles for 2 users.
4. Resolves #15

souravtecken committed:
Merge branch 'master' of https://github.com/varunvora/alcoding

souravtecken committed:
Merge branch 'master' of https://github.com/varunvora/alcoding

souravtecken committed:
Ran executor.sh for April Easy
